src/Button/Button.js

```diff
@@ -119,7 +119,7 @@ export const styles = theme => ({
     },
   },
   raisedAccent: {
-    color: theme.palette.getContrastText(theme.palette.secondary.light),
+    color: theme.palette.secondary.contrastText,
```

Comment:
This change is applying the contrastText color calculated for the main color to a light background. If the contrastText color is white, it's going to have a low contrast ratio. It also introduces an inconsistency with other components.

Reply:
@mbrookes Yes, it's the next topic I want to fix. We should be using main and darker. 
- https://material-components-web.appspot.com/button.html
- https://material.io/color/#!/?view.left=0&view.right=0
- https://material.io/guidelines/components/buttons.html#buttons-raised-buttons

Reply:
Actually, I was also going to submit an issue about why the default raisedAccent is not the main one and the hover the dark one. Is it what you are talking about @oliviertassinari